src/test/java/kitchenpos/application/MenuGroupServiceTest.java

```diff
@@ -0,0 +1,47 @@
+package kitchenpos.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+import kitchenpos.domain.MenuGroup;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.jdbc.Sql;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@Sql(scripts = {"classpath:truncate.sql"}, executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+class MenuGroupServiceTest {
+
+    @Autowired
+    private MenuGroupService menuGroupService;
+
+    @Test
+    void 메뉴그룹을_등록할_수_있다() {
+        // given
+        final MenuGroup menuGroup = new MenuGroup();
+        menuGroup.setName("테스트 메뉴 그룹");
+
+        // when
+        final MenuGroup result = menuGroupService.create(menuGroup);
+
+        // then
+        assertThat(result.getId()).isNotNull();
+        assertThat(result.getName()).isEqualTo(menuGroup.getName());
```

Comment:
```suggestion
        // when
        assertThat(menuGroup.getId()).isNull();
        final MenuGroup result = menuGroupService.create(menuGroup);

        // then
        assertThat(result.getId()).isNotNull();
        assertThat(result.getName()).isEqualTo(menuGroup.getName());
```

저장된 데이터인 `result.getId()`가 null이 아닌 것을 검증하려면, 저장되기 이전의 데이터가 null인 것을 한 번 검증해주어야 하지 않을까요 ?

Reply:
서비스 로직을 테스트 하는 입장에서 필요한 데이터를 세팅을 하게되는데,
그 세팅된 객체들을 테스트하지 않아도 괜찮다고 생각합니다.

// when 에서 발생하거나 반환되는 결과물들만,
테스트진행해도 충분할 것 같아요!!

Reply:
쥬니랑 얘기를 나눈 후
반환한 결과랑 디비에서 조회해온 결과를 비교하면 더 깔끔할 것 같아요!